vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -31338,3 +31278,1243 @@ THE UCUM TABLE (IN ALL FORMATS), UCUM DEFINITIONS, AND SPECIFICATION ARE PROVIDE
+  #  ──┬ unit:OZ_VOL_UK-PER-DAY multiplier: 0.00000787487 (correct: 0.0000000003288548900462962962962962962962962)
+  #    ├──┬ unit:DAY^-1 multiplier: 86400.0 (multiplier^-1: 0.00001157407407407407407407407407407407)
+  #    │  └─── unit:SEC multiplier: 1.0
+  #    └──┬ unit:OZ_VOL_UK multiplier: 0.0000284130625
+  #       └──┬ unit:M3 multiplier: 1.0
+  #          └─── unit:M^3 multiplier: 1.0 (multiplier^3: 1.0)
+unit:OZ_VOL_UK-PER-DAY qudt:conversionMultiplier 0.0000000003288548900462962962962962962962962 .
+
+  # unit:ANGSTROM3 has conversionMultiplier 1.0E-40, but we've found out that 1 Å³ = 1.000E-30 m³ (relative diff: 1.999999999600000000039999999996000 - big difference)
+  #  ──┬ unit:ANGSTROM3 multiplier: 0.0000000000000000000000000000000000000001 (correct: 0.000000000000000000000000000001)
+  #    └──┬ unit:ANGSTROM^3 multiplier: 0.0000000001 (multiplier^3: 0.000000000000000000000000000001)
+  #       └─── unit:M multiplier: 1.0
+unit:ANGSTROM3 qudt:conversionMultiplier 0.000000000000000000000000000001 .
+
+  # unit:W-M2-PER-SR has no conversionMultiplier, but we've found out that 1 W⋅m²/sr = 1 W⋅m²
+  #  ──┬ unit:W-M2-PER-SR multiplier: [no conversionMultiplier] (correct: 1.0)
+  #    ├─── unit:M^2 multiplier: 1.0 (multiplier^2: 1.0)
+  #    ├──┬ unit:SR^-1 multiplier: 1.0 (multiplier^-1: 1.0)
+  #    │  ├─── unit:M^-2 multiplier: 1.0 (multiplier^-2: 1.0)
+  #    │  └─── unit:M^2 multiplier: 1.0 (multiplier^2: 1.0)
+  #    └──┬ unit:W multiplier: 1.0
+  #       ├──┬ unit:J multiplier: 1.0
+  #       │  ├─── unit:M multiplier: 1.0
+  #       │  └──┬ unit:N multiplier: 1.0
+  #       │     ├──┬ unit:KiloGM multiplier: 1.0
+  #       │     │  └─── unit:GM multiplier: 0.001
+  #       │     ├─── unit:M multiplier: 1.0
+  #       │     └─── unit:SEC^-2 multiplier: 1.0 (multiplier^-2: 1.0)
+  #       └─── unit:SEC^-1 multiplier: 1.0 (multiplier^-1: 1.0)
+unit:W-M2-PER-SR qudt:conversionMultiplier 1.0 .
+
+  # unit:PER-WB has no conversionMultiplier, but we've found out that 1 /Wb = 1 A/J
+  #  ──┬ unit:PER-WB multiplier: [no conversionMultiplier] (correct: 1.0)
+  #    └──┬ unit:WB^-1 multiplier: 1.0 (multiplier^-1: 1.0)
+  #       ├─── unit:A^-1 multiplier: 1.0 (multiplier^-1: 1.0)
+  #       └──┬ unit:J multiplier: 1.0
+  #          ├─── unit:M multiplier: 1.0
+  #          └──┬ unit:N multiplier: 1.0
+  #             ├──┬ unit:KiloGM multiplier: 1.0
+  #             │  └─── unit:GM multiplier: 0.001
+  #             ├─── unit:M multiplier: 1.0
+  #             └─── unit:SEC^-2 multiplier: 1.0 (multiplier^-2: 1.0)
+unit:PER-WB qudt:conversionMultiplier 1.0 .
+
+  # unit:MilliR has conversionMultiplier 2.58E-7, but we've found out that 1 mR = 3.33564095198152049575576714474920E-13 C/kg (relative diff: 1.999994828470326527163271547239499 - big difference)
```

Comment:
This one seems wrong to me. Could you confirm?

Reply:
...and if it is wrong, all the units involving the roentgen need to be adjusted.

Reply:
I think you are right, good catch!

I've rebased onto main and removed the changes for R, MilliR and KiloR